utopia-remix/app/routes/projects.tsx

```diff
@@ -580,8 +603,10 @@ const NoProjectsMessage = React.memo(() => {
         return 'Public projects you own.'
```

Comment:
this can (theoretically) also be in the `categories` object descriptor at the top

Reply:
[`721ab83` (#5089)](https://github.com/concrete-utopia/utopia/pull/5089/commits/721ab83bd3fdc0636e8d018e3ed54c3d2f062d11)